pkg/TIPA.py

```
@@ -164,587 +191,489 @@ def __init__(self, img, mouse_pos, display_info_init, gui_queue):
 
-                        else:
-                            print("Request Gamepedia 200")
+                else:
+                    break
 
-                    except Exception as e:
-                        if self.debug_mode >= 1:
-                            print("Unexpected error:", "No Page Found for Gamepedia")
+            except Exception as e:
+                if tryCounter == 1:
+                    URL = f"https://tarkov-market.com/item/{corrected_text}".lower().capitalize()
```

Ruff (0.8.0)

Undefined name `corrected_text`

(F821)